lib/src/controller.dart

```diff
@@ -276,6 +276,11 @@ class MaplibreMapController extends ChangeNotifier {
     return _mapboxGlPlatform.animateCamera(cameraUpdate, duration: duration);
   }
 
+  Future<bool?> setPadding({required EdgeInsets edgeInsets}) async {
```

Comment:
Small rename to be consistent with the framework 👍🏽 
```suggestion
  Future<bool?> setPadding({required EdgeInsets padding}) async {
```